### PR TITLE
Arg viz

### PR DESCRIPTION
**Pull request by hyanwong**
Substantial rework of the ARG tube, to include visualisations

Also corrects a few typos in the forward sim tune from previously.

### CODE REVIEW

**Comment by hyanwong** [comment]
Hmm, pygraphviz install failing. I though I had installed the underlying C library using `apt-get`, but I probably missed something.

**Comment by hyanwong** [comment]
Ah - fixed now - needed the dev library and installing first.

File: args.md [hunk 1]
```diff
@@ -30,22 +25,25 @@ those nodes. Therefore a succinct tree sequence is equivalent to a
 which is additionally annotated with genomic positions such that at each
 position, a path through the edges exists which defines a tree. This graph
 interpretation of a tree sequence is tightly connected to the concept of
-an "ancestral recombination graph" (or ARG). 
+an "ancestral recombination graph" (or ARG). See
+[this preprint](https://www.biorxiv.org/content/10.1101/2023.11.03.565466v1) for further details.
```
Comment (benjeffery) [inline args.md:29]: Nice

**Review by benjeffery — APPROVED** [review]

**Comment by benjeffery** [comment]
Great stuff, builds and works locally and the HTML looks good. Merge away.

**Comment by hyanwong** [comment]
> Great stuff, builds and works locally and the HTML looks good

Great, thanks for looking over it.

> Merge away.

I don't have permissions to merge this: was this a comment to @jeromekelleher ?

**Pull request merged**